### PR TITLE
update crazybee.sh

### PR DESCRIPTION
**Pull request by BrunoLafleur**
Sync crazybee.sh from BumblebeeSlackbuilds

### CODE REVIEW

File: slackware/crazybee.sh [hunk 1]
```diff
@@ -89,7 +89,9 @@ install_latest_pkg_compat primus
 install_latest_pkg_compat primus_vk
 
 cd ../nouveau-blacklist/
-upgradepkg --reinstall xf86-video-nouveau-blacklist-noarch-1.txz
+removepkg xf86-video-nouveau-blacklist-noarch-1.txz
+ls -t --color=never xf86-video-nouveau-blacklist-*noarch-[0-9]*.txz | \
```
Comment (ryanpcmcquen) [inline slackware/crazybee.sh:93]: Let's use `find` instead, something like (untested idea):

```
find . -name "xf86-video-nouveau-blacklist-*noarch-[0-9]*.txz" -exec upgradepkg --install-new --reinstall {} \;
```

Comment (BrunoLafleur) [reply slackware/crazybee.sh:93]: The ls -t is here only for being sure we catch the latest xf86-video-nouveau-blacklist and not the old without 1.0 one.
It is the same trick which is used above in the script.
The find is useless here.

Comment (ryanpcmcquen) [reply slackware/crazybee.sh:93]: How about this?
```
find . -name "xf86-video-nouveau-blacklist-*noarch-[0-9]*.txz" -print0 | sort -z | head -1 | xargs -i upgradepkg --install-new --reinstall {}
```

Comment (BrunoLafleur) [reply slackware/crazybee.sh:93]: Why took a bomb to kill a bee ?
find is a heavier tool than ls.
The ls trick is good enough for what I want. It is also used above in the script (from the original authors). We can suppress the --color=never argument here and above in the install_latest_pkg* : 
ls -t xf86-video-nouveau-blacklist-*noarch-[0-9]*.txz | \
    head -1 | xargs -i upgradepkg --install-new --reinstall {}

Comment (ryanpcmcquen) [reply slackware/crazybee.sh:93]: Nah, I have had the `ls` colors bite me in this scenario before.

Comment (ryanpcmcquen) [reply slackware/crazybee.sh:93]: OK, well I would prefer `find` but I won't fight you on it.

**Pull request merged**